systemtest/010-inspect.bats

```diff
@@ -64,4 +64,19 @@ Os                 linux
 END_EXPECT
 }
 
+@test "inspect: env" {
+    remote_image=docker://docker.io/fedora:31
```

Comment:
Any way to make this not specific to a particular fedora version?  :latest?

Otherwise a year and a half from now this might start breaking or become very out of date.

Reply:
@rhatdan thanks, I wanna check special value of environment variable later, so a particular fedora version is given in test case, such as checking DISTTAG=f31container FGC=f31 FBR=f31

Reply:
Done